authApiServer/src/main/java/com/example/clubsite/controller/loginController.java

```diff
@@ -0,0 +1,132 @@
+package com.example.clubsite.controller;
+
+
+import com.example.clubsite.repository.ClubMemberRepository;
+import com.example.clubsite.security.service.ClubOAuth2UserDetailsService;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jose.shaded.json.JSONObject;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.*;
+import org.springframework.ui.Model;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.RestTemplate;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@RestController
+@Log4j2
+
+@RequiredArgsConstructor
+public class loginController {
```

Comment:
클래스 컨벤션은 맨 앞자 대문자입니다.
```suggestion
public class LoginController {
```